qermit/taskgraph/utils.py

```diff
@@ -177,7 +177,7 @@ class MeasurementCircuit(object):
     for some Ansatz Circuit.
     """
 
-    def __init__(self, symbolic_circuit: Circuit, symbols: SymbolsDict = None):
+    def __init__(self, symbolic_circuit: Circuit, symbols: Optional[SymbolsDict] = None):
```

Comment:
ah, is this what the newer mypy was picking up? Nice

Reply:
A couple of cases of optionals + signal_filter not defining a method as abstract